config/routes.rb

```diff
@@ -12,5 +12,11 @@
   end
   resources :presentations, except: [:edit]
 
+  namespace :admin do
```

Comment:
Re 246674828a043517ebbf980c27f19e709d7a2a9f commit message. We like to follow the pattern of "What changed, then why it changed". Anybody looking at the code of the commit can see the changes that were made. Something a little more ideal might look like:

```
Add admin namespaced People routes

We want to create an admin area where people's roles can be updated. We're going
to namespace this behind `/admin` to make it clear that this is a separate entity
from the main site.
```